src/Shopsys/ShopBundle/Resources/views/Front/Inline/MeasuringScript/googleAnalytics.html.twig

```diff
@@ -7,4 +7,26 @@
     ga('create', '{{ trackingId }}', 'auto');
     ga('send', 'pageview');
 
-</script>
\ No newline at end of file
+</script>
+
+{% if domainId == 1 %}
```

Comment:
I am no expert in this field, however, would not the new scripts somehow conflict with the script above? Or these are two different things? (https://www.analyticsmania.com/post/google-tag-manager-vs-google-analytics/)

Then, is not it confusing that GTM and GA are both in `googleAnalytics.html.twig` template? :innocent:

Reply:
I think they are not in conflict and I have split them as you expected